src/services/utilities.jsx

```diff
@@ -0,0 +1,27 @@
+export const fetchRequestByMethod = async (url, jsonBody, method ) => {
+    if( method.toUpperCase() === 'GET' || 'DELETE') {
```

Comment:
For the `||` here we need to check it like this `method.toUpperCase() === 'GET' || method.toUpperCase() === 'DELETE'` :)